packages/core/src/extensions/SuggestionMenu/getDefaultEmojiPickerItems.ts

```diff
@@ -0,0 +1,37 @@
+import data, { Emoji, EmojiMartData } from "@emoji-mart/data";
+import { init, SearchIndex } from "emoji-mart";
+
+import { BlockSchema, InlineContentSchema, StyleSchema } from "../../schema";
+import { BlockNoteEditor } from "../../editor/BlockNoteEditor";
+import { checkDefaultInlineContentTypeInSchema } from "../../blocks/defaultBlockTypeGuards";
+import { DefaultGridSuggestionItem } from "./DefaultGridSuggestionItem";
+
+const emojiMartData = data as EmojiMartData;
+init({ emojiMartData });
```

Comment:
quite a lot happens in this init function. I would only call this the first time `getDefaultEmojiPickerItems` is called